docs/syntax.rst

```diff
@@ -168,6 +168,35 @@ Type aliases create a new name for a type. It does not actually create a new typ
         type Age = int;
         type Coordinate = int[2];
 
+Generics/Generic Types
```

Comment:
Two more things:

1. add `flattenSha256(A a) : Sha256` into https://scryptdoc.readthedocs.io/en/latest/functions.html#hashing
2. Let us put all types into a separate file/subpage `types.rst`, now it grows big.

Reply:
2. The whole structure need some adjustments, let's do it in  https://github.com/sCrypt-Inc/specification/issues/10